policies/versioning-policy.md

```diff
@@ -0,0 +1,155 @@
+> version of _0_ and a _patch_ version of _1_.
+
+This closely aligns with the expectations of users who are familiar
+with _semantic versioning_.  However, it is not quite _semantic versioning_
+because adopting _semantic versioning_ would mean changing our current
+Long Term Support ([LTS]) policies and practices.
+
+## Major release: changing the first digit
+
+A major release can, and generally will, introduce API and ABI breaking
+changes.
+
+> For example, a program that runs with OpenSSL 3.0.1 should not expect to run
+> with OpenSSL 4.0.0 without modifications.  If it does run, it might not be
+> as efficient as it was.
+
+In order to maintain stability and limit rework across major versions:
+
+- Existing [public interfaces] will remain unmodified except where changes
+  are unlikely to break source compatibility or where structures are being
+  made opaque.
+- Existing public interfaces must remain in at least one LTS release
+  until the original interface has been documented as _deprecated_
+  for at least 5 years.
+- When structures are made opaque, any newly required accessor macros
+  or functions are added in a feature release of the extant LTS release
+  and all supported intermediate successor releases.
+
+Exceptions to these rules require a vote by the [OMC].  See also the
+[stable release update policy].
+
+## Minor release: changing the second digit
+
+A minor release can, and generally will, introduce new features.  However both
+the API and ABI will be preserved.
+
+> For example, a program built with OpenSSL release 3.0.1 will be able to
+> run with OpenSSL 3.1.0 but might not be able to take advantage of new
+> features without modification.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Patch release: changing the third digit
+
+A patch release will only contain bug fixes.  Both the API and ABI will remain
+compatible across patch releases.
+
+> For example, a program linked with OpenSSL release 3.0.0 can
+> run with OpenSSL 3.0.1 without changes.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Long term stable releases
+
+A release can be designated as a release.  LTS releases will be supported
+for at least five years and the project will specify a LTS release at
+least every four years.  Non-LTS releases will be supported for at least
+two years.
+
+During the final year of support, we do not commit to anything other
+than security fixes.  Before that, bug and security fixes will be applied
+as appropriate.
+
+The addition of new platforms to LTS branches is acceptable so long as
+the required changes consist solely of additions to configuration.
+
+## Prerelease versions
+
+Before a major release the project will generally make a number of pre-releases.
+These are labelled _alpha_ and _beta_ releases.
+
+### Alpha releases
+
+An _alpha_ release is one that is:
+
+- not necessarily feature complete and
+- not necessarily includes all new APIs.
```

Comment:
"and not necessarily includes" doesn't quite make sense.

Reply:
You want an Oxford semi-colon here?

Reply:
I would say:

````
- not necessarily feature complete and
- does not necessarily include all new APIs.
````

Reply:
Okay.  I pulled the prior _is_ into the first point for consistencty.